### PR TITLE
Compass class and test

### PR DESCRIPTION
**Pull request by sachsgit**
Just the new Compass Class and Unit Test for it

### CODE REVIEW

**Comment by codingricky** [comment]
Can you add it to the FakerIT please? Thanks

**Comment by sachsgit** [comment]
@codingricky I'm feeling a bit out of it, I cannot find a file called ".*FakerIT.*". 
Can you point it out to me?

**Comment by codingricky** [comment]
https://github.com/DiUS/java-faker/blob/8013b21745e67a7779e9c811f8bac35f81c67f95/src/test/java/com/github/javafaker/integration/FakerIT.java

**Comment by sachsgit** [comment]
@codingricky Done. "Added faker.compass() to FakeIT.java"

File: src/main/java/com/github/javafaker/Compass.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.github.javafaker;
+
+public class Compass {
+    private final Faker faker;
+
+    protected Compass(Faker faker) {
+        this.faker = faker;
+    }
+
+    public String directions() {
```
Comment (codingricky) [inline src/main/java/com/github/javafaker/Compass.java]: be good to make that consistent with the string key in the yml file. Maybe go with the singular form, so "direction"

Comment (sachsgit) [reply src/main/java/com/github/javafaker/Compass.java]: Missed that. Fixed.

**Comment by sachsgit** [comment]
@codingricky Please review for merging into master.

**Comment by codingricky** [comment]
Too many other files changed, can't merge

**Pull request closed without merging**